homeassistant/components/light/yeelight.py

```diff
@@ -195,6 +206,15 @@ def __init__(self, device, config):
         self._is_on = None
         self._hs = None
 
+        self._model = config['model']
+        self._min_mireds = None
```

Comment:
Maybe move the `kelvin_to_mired(YEELIGHT_SPECS['color1']['max_kelvin'])` defaults from `def min_mireds` here?

Reply:
I could move it to the update call. The capabilities of the device are determined at runtime unfortunately.